JustReleaseNotes/factory.py

```diff
@@ -10,6 +10,15 @@ def createWithConfig(module, name, conf):
         e = sys.exc_info()[1]
         raise Exception("Unable to create module provider {0} from module {1}: {2}".format(conf["Provider"], module, e))
 
+def createWithConfigAndParam(module, name, conf, param):
```

Comment:
Do we need this ? Can't the writer just extract the right setting from the config ?